packages/cli/src/services/config-service.ts

```diff
@@ -35,23 +35,29 @@ async function compileProject(projectPath: string): Promise<void> {
 }
 
 function readProjectConfig(userProjectPath: string): Promise<BoosterConfig> {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
   const userProject = loadUserProject(userProjectPath)
   return new Promise((resolve): void => {
-    const app: BoosterApp = userProject.Booster
+    const app = userProject.Booster
     app.configureCurrentEnv((config: BoosterConfig): void => {
       checkEnvironmentWasConfigured(app)
       resolve(config)
     })
   })
 }
 
-function loadUserProject(userProjectPath: string): { Booster: BoosterApp } {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function loadUserProject(userProjectPath: string): { Booster: any } {
```

Comment:
Why changing this to `any`? On the calling site, we now have lost compile-time guarantees. 

For example, 9 lines above, on line 41 of this file, we are calling `app.configureCurrentEnv`, but that call is not checked at compile time now. If we change the name of the function or its parameters, it will silently compile but blow up in runtime.

Reply:
This is the first change towards the refactorization described in #517, so these `any` references will eventually disappear. After this change, `BoosterApp` is no longer part of the types package, so I have no easy way to require it.